e2e/cypress/integration/enterprise/cloud/Billing/company_Information_spec.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only
+
+import {getRandomLetter} from '../../../../utils/index';
+
+describe('System Console - Company Information section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-37051 - Adding the Company Information screen', () => {
+        const companyName = getRandomLetter(30);
+        gotoCompanyInformationScreen();
+
+        // # click on Add Company Information button
+        cy.contains('span', 'Add Company Information').parent().click();
+
+        // # enter company information functionality
+        cy.get('#input_companyName').clear().type(companyName);
+        cy.get('#input_numEmployees').clear().type('10');
+        cy.get('#DropdownInput_country_dropdown').click();
+        cy.get(".DropDown__input > input[type='text']").type('India{enter}');
+        cy.get('#input_address').clear().type('Add test address');
+        cy.get('#input_address2').clear().type('Add test address2');
+        cy.get('#input_city').clear().type('Addtestcity');
+        cy.get('#input_state').clear().type('Addteststate');
+        cy.get('#input_postalCode').clear().type('560089');
+
+        // # click on Save Info button
+        cy.get('#saveSetting').should('be.enabled').click();
```

Comment:
Could we add a test that validates the save button is disabled when fields are incomplete?

Reply:
added new test case in the script for checking disabled status of save button